src/core/api/subscription.ts

```diff
@@ -0,0 +1,14 @@
+import { apiClient } from '~core/apiClientInstance';
+import type { CurrentSubscriptionInfo } from '~features/subscriptions/types';
+
+export async function getCurrentUserSubscription() {
+  try {
+    return await apiClient.get<CurrentSubscriptionInfo>(
+      '/current_user/subscription ',
+      undefined,
+      true,
+    );
+  } catch (error) {
+    return null;
```

Comment:
are you sure about returning just null in case of error?

Reply:
This was handled like this, because at first if there is no data to return about current subscription, server responded with 404 error. Now we decided to change it and it will be 204 response. I will make appropriate changes here as well